dot-studio/golang

```diff
@@ -57,13 +57,28 @@ function setup_go_workspace() {
     fi;
   fi;
 
+  # Install go & git
+  install_if_missing core/go go;
+  install_if_missing core/git git;
+
   echo "=> Available scaffolding variables:";
   echo "\$scaffolding_go_gopath        - $scaffolding_go_gopath";
   echo "\$scaffolding_go_workspace_src - $scaffolding_go_workspace_src";
   echo "\$scaffolding_go_pkg_path      - $scaffolding_go_pkg_path";
   return 0;
 }
 
+document "enter_go_workspace" <<DOC
+  Change dir to the $pkg_name Go Workspace.
+
+  It will automatically setup the Go Workspace if missing
+DOC
+function enter_go_workspace() {
+  setup_go_workspace
+  cd $scaffolding_go_pkg_path
+}
+add_alias "enter_go_workspace" "ew"
```

Comment:
what if we wanted to enter other types of workspaces? do we want to make this alias `egw`?

Reply:
Totally agreeeee!!! I was going to ask you about aliases that are more than 2 letters. Will do the change!